proto/buf/alpha/registry/v1alpha1/resolve.proto

```diff
@@ -107,3 +111,24 @@ message GetLocalModulePinsResponse {
   // This includes the transitive deps.
   repeated buf.alpha.module.v1alpha1.ModulePin dependencies = 2;
 }
+
+message ResolveReferenceRequest {
+  PluginRef plugin = 1;
+  ModuleRef module = 2;
+}
+
+message PluginRef {
```

Comment:
We have a lot of representations for these, is there a reason these are special and aren't reused? Is it because of `remote` (which makes sense)

Reply:
Yeah